budgetportal/urls.py

```diff
@@ -1,6 +1,10 @@
 from adminplus.sites import AdminSitePlus
 
-from budgetportal.views import openspending_csv, about, infrastructure_project_list
+from budgetportal.views import openspending_csv, about, events, videos, terms_and_conditions, search_result, resources, \
```

Comment:
looks like budgetportal.views is already imported as `views` so let's just refer to these in the rest of this module under `views.`